src/test/java/ca/gc/aafc/collection/api/entities/MaterialSampleCRUDIT.java

```diff
@@ -9,64 +9,91 @@
 
 @SpringBootTest(properties = "keycloak.enabled=true")
 
 public class MaterialSampleCRUDIT extends CollectionModuleBaseIT {
 
     @Inject
-    private DatabaseSupportService dbService;
+    private PreparationTypeService preparationTypeService;
+
+    @Inject
+    private MaterialSampleService materialSampleService;
 
     private List<UUID> attachmentIdentifiers = List.of(UUID.randomUUID(), UUID.randomUUID());
 
-    @Test
-    public void testSave() {
-        String dwcCatalogNumber = "S-4313";
-        String expectedCreatedBy = "dina-save";
-        String sampleMaterialName = "lake water sample";
+    private static final String dwcCatalogNumber = "S-4313";
+    private static final String expectedCreatedBy = "dina-save";
+    private static final String sampleMaterialName = "lake water sample";
+
+    private static final String preparationExpectedCreatedBy = "preparation-dina-save";
+    private static final String preparationExpectedGroup = "dina-group-save";
+    private static final String preparationExpectedName = "isolate lake water sample";
+
+    private PreparationType preparationType;
+    private MaterialSample materialSample;
 
-        MaterialSample  materialSample = MaterialSampleFactory.newMaterialSample()
+    @BeforeEach
+    void setup() {
+        preparationType = PreparationTypeFactory.newPreparationType()
+            .createdBy(preparationExpectedCreatedBy)
+            .group(preparationExpectedGroup)
+            .name(preparationExpectedName)
+            .build();
+        preparationTypeService.create(preparationType);
+
+        materialSample = MaterialSampleFactory.newMaterialSample()
             .dwcCatalogNumber(dwcCatalogNumber)
             .createdBy(expectedCreatedBy)
             .attachment(attachmentIdentifiers)
             .materialSampleName(sampleMaterialName)
+            .preparationType(preparationType)
             .build();
-        assertNull(materialSample.getId());
-        dbService.save(materialSample);
+        materialSampleService.create(materialSample);
+    }
+
+    @Test
+    public void testCreate() {
+
         assertNotNull(materialSample.getId());
+
         assertEquals(dwcCatalogNumber, materialSample.getDwcCatalogNumber());
         assertEquals(expectedCreatedBy, materialSample.getCreatedBy());
         assertEquals(attachmentIdentifiers, materialSample.getAttachment());
         assertEquals(sampleMaterialName, materialSample.getMaterialSampleName());
+        assertNotNull(preparationType.getId());
+        assertEquals(preparationType.getId(), materialSample.getPreparationType().getId());
+        assertEquals(preparationExpectedCreatedBy, materialSample.getPreparationType().getCreatedBy());
+        assertEquals(preparationExpectedGroup, materialSample.getPreparationType().getGroup());
+        assertEquals(preparationExpectedName, materialSample.getPreparationType().getName());
     }
 
     @Test
     public void testFind() {
```

Comment:
we should re-evaluate what we are asserting in this method. this is largely a copy of test create, the fetchedMaterialSample seems to not be asserted on as it should